modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -4279,32 +4279,42 @@ definition {
 	@description = "Verify that the Is Not Equals to operator is available when adding filters for the integer fields to select data from the object"
 	@priority = 4
 	test IsEqualToIsAvailableForIntegerFields {
-		task ("Given a custom object with an integer field type") {
+		task ("Given the custom objects with relationship") {
 			ObjectAdmin.addObjectViaAPI(
```

Comment:
In this case, instead of using `the`, you can just specify how many are created. So you can say: `Given two object definitions with an integer field and a relationship`. 

You can also split them into two different givens if you want:
- Given two object definitions, Custom Object A and Custom Object B
- and Given an integer field in Custom Object A and a relationship between the two object definitions